android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -576,6 +576,21 @@
 	/**
 	 * @module.api
 	 */
+	public static final String EVENT_DRAWER_CLOSE = "drawerclose";
+
+	/**
+	 * @module.api
+	 */
+	public static final String EVENT_DRAWER_OPEN = "draweropen";
+
+	/**
+	 * @module.api
+	 */
+	public static final String EVENT_DRAWER_SLIDE = "drawerslide";
```

Comment:
Can we name it `open`, `close` and `slide` instead? Feels more unified with other related API's. Thx!